helpers/CssHelper.php

```diff
@@ -26,25 +26,7 @@ class CssHelper{
 
     private static function _buildWarning() {
         return <<<EOF
```

Comment:
Since this is now a on liner there is no point in using [heredoc ](http://php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc) anymore. Please use `return '/* Do not edit */';` instead.

Reply:
This method is used twice in this class. You can create a constant for this string and use it. but it will be the same as leaving the method.